test/config/config.go

```diff
@@ -26,3 +26,12 @@ func GetBaseURL() string {
 	}
 	return "http://localhost:8081"
 }
+
+func GetBaseURL2() string {
```

Comment:
I'm not sure about this function. What are your thoughts about putting the whole environment setup logic in a single `GetBaseURL()` function instead?